src/net/FetchHttpClient.ts

```diff
@@ -11,6 +12,9 @@ const {HttpClient, HttpClientResponse} = require('./HttpClient');
  * node-fetch package (https://github.com/node-fetch/node-fetch).
  */
 class FetchHttpClient extends HttpClient {
+  _fetchFn: any;
```

Comment:
Would `(input: RequestInfo | URL, init?: RequestInit | undefined) => Promise<Response>` work? Or at least some kind of lambda.

Reply:
Yep, that works! Thank you 🙌